README.md

```diff
@@ -1 +1,47 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is a project that I learned about linter, HTML and css
+
```

Comment:
It's gonna be better if you improve your description if you want to have a reference, 3 lines or more would be enough, try to tell about what you did, what kind of technologies you used, what is the mean scope for this project, and so on.

- [ ]  Improve your project description.